src/i18n.ts

```diff
@@ -195,6 +197,7 @@ should either enable sessions or use `ctx.i18n.useLocale()` instead.",
 
     ctx.i18n = {
       fluent,
+      locale,
```

Comment:
This won't update automatically when the variable is overwritten, you'd have to assign ctx.i18n.locale, as well. Alternatively, your could use a getter.

Reply:
Thanks, I'll use getter.